src/main/java/dk/cngroup/lentils/service/CypherService.java

```diff
@@ -94,8 +94,7 @@ public Cypher getFirstOrderByStageAsc() {
 
     @Transactional
     public void deleteById(final Long id) {
-        getCypher(id).getHints().stream()
-                .forEach(hint -> hintService.deleteById(hint.getHintId()));
+        getCypher(id).getHints().forEach(hint -> hintService.deleteById(hint.getHintId()));
```

Comment:
Můžu se zeptat, proč tady rušíš ten stream? Souvisí to s RestControllerem, nebo spíš jen protože je to tam zbytečné? 
Dík

Reply:
Jen protože to tam je zbytečně :) všiml jsem si toho tak jsem to dal společně do tohoto pullrequestu.